### ISSUE TITLE
Mana blaster beam has annoying green lines at the right side

### ISSUE THREAD

**Issue by Acat1234** [Vazkii/Botania#4640]
### Mod Loader

Forge

### Minecraft Version

1.20.1

### Botania version

443

### Modloader version

47.2.32

### The latest.log file

Idk what this means

### Issue description

It has a annoying green line at the right side and I cant fix it here is what it looks like
![2024-05-10_17 15 32](https://github.com/VazkiiMods/Botania/assets/126712560/8f22fea2-1ac8-4700-a153-00fa65d3b8be)

**Comment by Acat1234** [comment]
Oh and can it be held better like it is being aimed and not swung and not swinging to use?

**Comment by Acat1234** [comment]
And it seams to happen for mana pool particles too

**Comment by Acat1234** [comment]
Also seams to happen with the teleport eye relic

**Comment by TheRealWormbo** [comment]
Yes, those are all the same particle texture, since Botania for the most part restricts itself to only two of those. I'm not sure what causes them, but the fact that you need a relatively large particle seems to point towards mip-mapping issues, potentially a texture atlas thing.

**Comment by Acat1234** [comment]
The latest version makes the line on the top now
